hack/make/tests/e2e.mk

```diff
@@ -8,18 +8,20 @@ test/e2e/%/publish:
 test/e2e/%/debug:
 	@make SKIPCLEANUP="--fail-fast" $(@D)
 
-## Run standard, istio and release e2e tests
+## Run standard, no-csi, istio and release e2e tests
```

Comment:
the only comment - i have is: here we use `noCSI`, but later in code itself it's `withoutCSI`.
I personally prefer last one (withoutCSI), but it's up to you.